### PR TITLE
dxFileUploader: describe upload methods

### PR DESCRIPTION
**Pull request by AnnaKondratova**

### CODE REVIEW

File: api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
 ##### shortDescription
-<!-- Description goes here -->
+This section describes methods that control the **FileUploader** widget.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: What do you mean by "that control"? Control it how?

File: api-reference/10 UI Widgets/dxFileUploader/3 Methods/upload().md [hunk 2]
```diff
@@ -3,7 +3,10 @@ id: dxFileUploader.upload()
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Uploads all selected files.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxFileUploader/3 Methods/upload().md]:
```suggestion
Uploads all the selected files.
```

File: api-reference/10 UI Widgets/dxFileUploader/3 Methods/upload().md [hunk 3]
```diff
@@ -3,7 +3,10 @@ id: dxFileUploader.upload()
...
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+
+[note]
+File upload is not supported in **useForms** upload mode ([uploadMode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode)). In **instantly** upload mode, use [value](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#value) to select a file(s) for uploading.
```
Comment (dirk-pieterse) [inline api-reference/10 UI Widgets/dxFileUploader/3 Methods/upload().md]:
```suggestion
File upload is not supported in **useForms** upload mode ([uploadMode](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#uploadMode)). In **instantly** upload mode, use [value](/Documentation/ApiReference/UI_Widgets/dxFileUploader/Configuration/#value) to select a file(s) you want to upload.
```

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md (hunk 1 above)
Comment (AnnaKondratova) [reply api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: It can by any actions. Not the file uploader has only upload methods.

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: You should add more information

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md (hunk 1 above)
Comment (AnnaKondratova) [reply api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: All 'Methods' topics in the DevExtreme documentation has the same description. 

There is no need to list all functionality as this description is followed by all widget's methods at the same page in the documentation.

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: Ok, but it does not explain what you mean when you say it 'controls' the widget. What does 'control' imply? How does it control the widget? What does it do?

**Review by dirk-pieterse — APPROVED** [review]

File: api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md (hunk 1 above)
Comment (AnnaKondratova) [reply api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: It means that methods allow you to manage the widget. May I use:

> This section describes methods that manage the **FileUploader** widget.

Comment (dirk-pieterse) [reply api-reference/10 UI Widgets/dxFileUploader/3 Methods/Methods.md]: 'manage', like 'control', do not explain what these methods do. How do they manage or control the widget? What do these methods do? If you say, "John manages a grocery store", you can ask, okay, what does he actually do? He supervises the staff in the store, he arranges deliveries of new goods, he ensures the accounting records are accurate, etc.

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**